services/github/github.go

```diff
@@ -50,17 +51,20 @@ var ownerRepoRegex = regexp.MustCompile(`^([A-z0-9-_.]+)/([A-z0-9-_.]+)$`)
 //
 // Before you can set up a Github Service, you need to set up a Github Realm.
 //
-// You can set a "default repository" for a Matrix room by sending a `m.room.bot.options` state event
+// You can set optional config for a Matrix room by sending a `m.room.bot.options` state event
 // which has the following `content`:
 //
 //  {
 //    "github": {
-//      "default_repo": "owner/repo"
+//      // The default repository to use for this room; this allows "owner/repo" to be omitted
+//      // when creating/expanding issues.
+//      "default_repo": "owner/repo",
+//
+//      // Comma-separated Github labels to attach to any issue created by this bot in this room.
+//      "new_issue_labels": "bot-label-1, bot-label-2"
```

Comment:
it should be possible to define this as an array, then parse it as an array magically in one of the layers. If there's something that's causing that to not work, a comment in `newIssueLabels` would be appreciated for documentation purposes.

Reply:
+1, especially if you are a dodgy person who puts commas in their issue labels.